src/components/app/ProfileViewer.css

```diff
@@ -11,4 +11,31 @@
   background-color: #fff;
   animation-name: profileViewerFadeIn;
   animation-duration: 200ms;
-}
\ No newline at end of file
+}
+
+.profileViewerName {
+  height: 24px;
+  padding: 0 6px 0 2px;
+  border-right: 1px solid var(--grey-30);
+  line-height: 24px;
+  font-weight: bold;
+  color: var(--grey-60);
+}
+
+.profileViewerZipButton {
+  width: 18px;
+  height: 18px;
+
+  margin: 3px;
+  padding: 0;
+  border: 1px solid var(--green-60);
+
+  color: #000;
+  background-color: var(--green-50);
+  border-radius: 3px;
+
+}
```

Comment:
The "arrow" isn't well centered on my computer, see 
![image](https://user-images.githubusercontent.com/454175/37660321-6de00db4-2c52-11e8-97f2-871e3df99310.png)

(I'm fine fixing this in a follow-up)

I believe part of the problem is that the font we use is different on each platform (I think it's "message-box" which is platform-dependent), but for emoji-like character we should force a specific font. Or we could use a simple SVG instead.

Reply:
Good point: #877